velox/connectors/lakehouse/iceberg/ConnectorConfigBase.cpp

```diff
@@ -16,9 +16,7 @@
 
```

Comment:
The file names should not be ConnectorXXX any more. They need to be called IcebergXXX